src/wayland-eglsurface.c

```diff
@@ -1206,7 +1206,9 @@ wlEglSurfaceCheckReleasePoints(WlEglDisplay *display, WlEglSurface *surface)
                            DRM_SYNCOBJ_WAIT_FLAGS_WAIT_AVAILABLE,
                            &firstSignaled) != 0) {
         /* A timeout is the only type of error we expect here */
+#ifdef __linux__
```

Comment:
Rather than `#ifdef __linux__`, we should just check `#ifdef ETIME` directly.

Reply:
Updated